app/controllers/users/delete_controller.rb

```diff
@@ -52,13 +53,22 @@ def send_push_notifications
       PushNotification::HttpPush.deliver(event)
     end
 
-    # rubocop:disable IdentityIdp/MailLaterLinter
     def notify_user_via_email_of_deletion
       current_user.confirmed_email_addresses.each do |email_address|
         UserMailer.with(user: current_user, email_address: email_address).
           account_delete_submitted.deliver_now
       end
     end
-    # rubocop:enable IdentityIdp/MailLaterLinter
```

Comment:
Based on build failure, I assume you're going to have to add back the Rubocop inline configuration comments.